op-deployer/pkg/deployer/bootstrap/bootstrap.go

```diff
@@ -73,12 +68,11 @@ func OPCMCLI(cliCtx *cli.Context) error {
 
 	l1RPCUrl := cliCtx.String(deployer.L1RPCURLFlagName)
 	privateKey := cliCtx.String(deployer.PrivateKeyFlagName)
-	artifactsURLStr := cliCtx.String(ArtifactsURLFlagName)
+	artifactsURLStr := cliCtx.String(ArtifactsLocatorFlagName)
```

Comment:
if the flag isn't supplied, should we try reading the intent?

Reply:
No - the bootstrap command is designed to work without the intent, since its purpose is to create a shared OPCM for use with other pipeline.